analytics/private-cloud/quotaChanges.ts

```diff
@@ -0,0 +1,95 @@
+import { PrivateCloudRequestedProject, DecisionStatus } from '@prisma/client';
+import prisma from '@/lib/prisma';
+
+export type DataPoint = {
+  date: string;
+  'Quota requests': number;
+};
+
+function createMonthKey(date: Date) {
+  const formatter = new Intl.DateTimeFormat('en', { month: 'short', year: '2-digit' });
+  return formatter.format(date);
+}
+
+export function isQuotaChanged(
+  projectOne: PrivateCloudRequestedProject,
+  projectTwo: PrivateCloudRequestedProject,
+): boolean {
+  // Assuming productionQuota, testQuota, developmentQuota, and toolsQuota are defined and comparable
+  return !(
+    JSON.stringify(projectOne.productionQuota) === JSON.stringify(projectTwo.productionQuota) &&
```

Comment:
maybe we can try `lodash` `isEqual` function to compare two objects?
```sh
import _isEqual from 'lodash-es/isEqual';
_isEqual(projectOne.productionQuota, projectTwo.productionQuota)
```

Note that I forgot to install `types` when installing `lodash-es`, please install it
```sh
npm i --save-dev @types/lodash-es
```